demo/grid.html

```diff
@@ -187,7 +187,7 @@
           <div class="photoContainer">
             <div class="photoContent" tabindex$="[[tabIndex]]">
               <iron-image sizing="cover"
-                  src="//farm[[photo.farm]].staticflickr.com/[[photo.server]]/[[photo.id]]_[[photo.secret]].jpg">
+                  src="//farm[[photo.farm]].staticflickr.com/[[photo.server]]/[[photo.id]]_[[photo.secret]]_n.jpg">
```

Comment:
Showed this demo to a friend (@spacerockzero) who pointed out we weren't using the Size Suffixes. Just adding this saved us `1.3mb`